### PR TITLE
Update spring-boot.md

### PR DESCRIPTION
**Pull request by steverao**
Polished the document.

### CODE REVIEW

**Review by svrnm — APPROVED** [review]

File: content/en/docs/languages/java/automatic/spring-boot.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ with the OpenTelemetry
 
 Autoconfigures OpenTelemetry instrumentation for
 [spring-web](#spring-web-autoconfiguration) ,
```
Comment (chalin) [inline content/en/docs/languages/java/automatic/spring-boot.md:234]:
```suggestion
[spring-web](#spring-web-autoconfiguration),
```

File: content/en/docs/languages/java/automatic/spring-boot.md [hunk 2]
```diff
@@ -232,7 +232,7 @@ with the OpenTelemetry
...
-[spring-webmvc](#spring-web-mvc-autoconfiguration), and
+[spring-webmvc](#spring-web-mvc-autoconfiguration) and
```
Comment (chalin) [inline content/en/docs/languages/java/automatic/spring-boot.md:235]:
```suggestion
[spring-webmvc](#spring-web-mvc-autoconfiguration), and
```

**Review by breedx-splk — APPROVED** [review]
Agree about the commas, fine otherwise. Thanks.

**Review by trask — APPROVED** [review]

**Review by cartermp — APPROVED** [review]

**Review by chalin — APPROVED** [review]
Thanks for submitting these copyedits @steverao.

**Pull request merged**